templates/package/content/container.tmpl

```diff
@@ -36,11 +36,13 @@
 				</thead>
 				<tbody>
 					{{range .PackageDescriptor.Metadata.Manifests}}
-					<tr>
-						<td><a href="{{$.PackageDescriptor.PackageWebLink}}/{{PathEscape .Digest}}">{{.Digest}}</a></td>
-						<td>{{.Platform}}</td>
-						<td>{{FileSize .Size}}</td>
-					</tr>
+						{{if ne .Platform "unknown/unknown"}}
+						<tr>
+							<td><a class="ui sha label" href="{{$.PackageDescriptor.PackageWebLink}}/{{PathEscape .Digest}}">{{ShortSha256 .Digest}}</a></td>
```

Comment:
If you truncate the digest, how could users get the full digest string?

Reply:
How about adding a copy button？

Reply:
Hmm, seems no problem since it is a link, users could open the link and copy the full digest.